examples/client_connectivitycheck_loop.c

```diff
@@ -48,6 +48,9 @@ int main(void) {
     /* Perform a connectivity check every 2 seconds */
     config.connectivityCheckInterval = 2000;
 
+    /* Client run in asynchronous mode */
+    config.asyncMode = true;
```

Comment:
From the test performed in ua_client.c, it should be enough to have connection->pendingMessage.
So we don't need to change the config API.

If we need asyncMode, I would move it to the client struct and not to the public config.

Reply:
No really needed. Only used to remove duplicates checks. Ok I remove the code.